changelog.d/11593.bugfix

```diff
@@ -0,0 +1 @@
+Fix an error to get federation status of a destination server even if no error has occurred.
```

Comment:
Can you include when the bug was introduced and note somewhere that this only applies to an admin API? Thanks!